Get Errors when try to 'make'

### Expected behavior

Make runs through its process without any errors

### Actual behavior

When execute make make -j$(nproc), the following output is produced:

`root@srvgvm01:/home/xxx/build/gvmd# make -j$(nproc)`
`[  0%] Building C object src/CMakeFiles/gvm-pg-server.dir/manage_pg_server.c.o`
`[  5%] Building C object src/CMakeFiles/gvm-pg-server.dir/manage_utils.c.o`
`[  5%] Generating gmp.rnc`
`[  5%] Generating gmp.html`
`[ 11%] Building C object src/CMakeFiles/gvmd.dir/gvmd.c.o`
`[ 11%] Building C object src/CMakeFiles/gvmd.dir/main.c.o`
`[ 11%] Building C object src/CMakeFiles/gvmd.dir/debug_utils.c.o`
`[ 16%] Building C object src/CMakeFiles/gvmd.dir/gmpd.c.o`
`[ 16%] Building C object src/CMakeFiles/gvmd.dir/manage_utils.c.o`
`[ 16%] Building C object src/CMakeFiles/gvmd.dir/manage.c.o`
`[ 22%] Building C object src/CMakeFiles/gvmd.dir/sql.c.o`
`[ 22%] Building C object src/CMakeFiles/gvmd.dir/manage_acl.c.o`
`[ 27%] Building GMP documentation...`
`[ 27%] Building C object src/CMakeFiles/gvmd.dir/manage_configs.c.o`
`[ 27%] Built target doc-gmp`
`[ 33%] Building C object src/CMakeFiles/gvmd.dir/manage_get.c.o`
`[ 33%] Linking C shared library libgvm-pg-server.so`
`[ 33%] Building C object src/CMakeFiles/gvmd.dir/manage_license.c.o`
`[ 38%] Building C object src/CMakeFiles/gvmd.dir/manage_port_lists.c.o`
`[ 38%] Building C object src/CMakeFiles/gvmd.dir/manage_preferences.c.o`
`[ 38%] Built target gvm-pg-server`
`[ 38%] Building C object src/CMakeFiles/gvmd.dir/manage_report_formats.c.o`
`[ 44%] Building C object src/CMakeFiles/gvmd.dir/manage_authentication.c.o`
`[ 44%] Building C object src/CMakeFiles/gvmd.dir/manage_sql.c.o`
`[ 44%] Building C object src/CMakeFiles/gvmd.dir/manage_sql_nvts.c.o`
`[ 50%] Building C object src/CMakeFiles/gvmd.dir/manage_sql_secinfo.c.o`
`[ 50%] Building C object src/CMakeFiles/gvmd.dir/manage_sql_port_lists.c.o`
`[ 55%] Building C object src/CMakeFiles/gvmd.dir/manage_sql_configs.c.o`
`[ 55%] Building C object src/CMakeFiles/gvmd.dir/manage_sql_report_formats.c.o`
`[ 55%] Building C object src/CMakeFiles/gvmd.dir/manage_sql_tickets.c.o`
`/home/xxx/source/gvmd-22.8.0/src/manage_sql.c: In function ‘credentials_setup’:`
`/home/xxx/source/gvmd-22.8.0/src/manage_sql.c:17195:14: error: ‘credentials_t’ has no member named ‘excerpt_size’`
`17195 |   credentials->excerpt_size`
      `|              ^~`
`[ 61%] Building C object src/CMakeFiles/gvmd.dir/manage_sql_tls_certificates.c.o`
`[ 61%] Building C object src/CMakeFiles/gvmd.dir/manage_tls_certificates.c.o`
`[ 61%] Building C object src/CMakeFiles/gvmd.dir/manage_migrators.c.o`
`[ 66%] Building C object src/CMakeFiles/gvmd.dir/sql_pg.c.o`
`[ 66%] Building C object src/CMakeFiles/gvmd.dir/manage_pg.c.o`
`[ 72%] Building C object src/CMakeFiles/gvmd.dir/lsc_user.c.o`
`[ 72%] Building C object src/CMakeFiles/gvmd.dir/lsc_crypt.c.o`
`[ 72%] Building C object src/CMakeFiles/gvmd.dir/utils.c.o`
`[ 77%] Building C object src/CMakeFiles/gvmd.dir/gmp.c.o`
`[ 77%] Building C object src/CMakeFiles/gvmd.dir/gmp_base.c.o`
`[ 77%] Building C object src/CMakeFiles/gvmd.dir/gmp_configs.c.o`
`[ 83%] Building C object src/CMakeFiles/gvmd.dir/gmp_delete.c.o`
`[ 83%] Building C object src/CMakeFiles/gvmd.dir/gmp_get.c.o`
`[ 88%] Building C object src/CMakeFiles/gvmd.dir/gmp_license.c.o`
`[ 88%] Building C object src/CMakeFiles/gvmd.dir/gmp_logout.c.o`
`[ 88%] Building C object src/CMakeFiles/gvmd.dir/gmp_port_lists.c.o`
`[ 94%] Building C object src/CMakeFiles/gvmd.dir/gmp_report_formats.c.o`
`[ 94%] Building C object src/CMakeFiles/gvmd.dir/gmp_tickets.c.o`
`[ 94%] Building C object src/CMakeFiles/gvmd.dir/gmp_tls_certificates.c.o`
`/home/xxx/source/gvmd-22.8.0/src/manage_sql.c: In function ‘setting_excerpt_size_int’:`
`/home/xxx/source/gvmd-22.8.0/src/manage_sql.c:50196:26: error: ‘credentials_t’ has no member named ‘excerpt_size’`
`50196 |   if (current_credentials.excerpt_size <= 0)`
      `|                          ^`
`/home/xxx/source/gvmd-22.8.0/src/manage_sql.c:50198:29: error: ‘credentials_t’ has no member named ‘excerpt_size’`
`50198 |   return current_credentials.excerpt_size;`
      `|                             ^`
`/home/xxx/source/gvmd-22.8.0/src/manage_sql.c: In function ‘modify_setting’:`
`/home/xxx/source/gvmd-22.8.0/src/manage_sql.c:50659:30: error: ‘credentials_t’ has no member named ‘excerpt_size’`
`50659 |           current_credentials.excerpt_size = atoi (value);`
      `|                              ^`
`/home/xxx/source/gvmd-22.8.0/src/manage_sql.c: In function ‘setting_excerpt_size_int’:`
`/home/xxx/source/gvmd-22.8.0/src/manage_sql.c:50199:1: warning: control reaches end of non-void function [-Wreturn-type]`
`50199 | }`
      `| ^`
`make[2]: *** [src/CMakeFiles/gvmd.dir/build.make:286: src/CMakeFiles/gvmd.dir/manage_sql.c.o] Error 1`
`make[2]: *** Waiting for unfinished jobs....`
`make[1]: *** [CMakeFiles/Makefile2:336: src/CMakeFiles/gvmd.dir/all] Error 2`
`make: *** [Makefile:166: all] Error 2`

### Steps to reproduce

Followed the Greenbone "Install from Source" Guide
https://greenbone.github.io/docs/latest/22.4/source-build/index.html#gvmd

### GVM versions

gvm: 22.8.0

gvm-libs: 22.6.3

### Environment

Operating system: Debian 12 Bookworm

root@srvgvm01:/home/hfp/build/gvmd# uname -a
Linux srvgvm01 6.1.0-11-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.1.38-4 (2023-08-08) x86_64 GNU/Linux

Installation method / source: source installation

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -26,7 +26,7 @@ find_package (Threads)
 ## list and throw an error, otherwise long install-cmake-install-cmake cycles
 ## might occur.
 
-pkg_check_modules (LIBGVM_BASE REQUIRED libgvm_base>=22.6)
+pkg_check_modules (LIBGVM_BASE REQUIRED libgvm_base>=22.7)
 pkg_check_modules (LIBGVM_UTIL REQUIRED libgvm_util>=22.6)
 pkg_check_modules (LIBGVM_OSP REQUIRED libgvm_osp>=22.6)
 pkg_check_modules (LIBGVM_GMP REQUIRED libgvm_gmp>=22.6)

